gradle.properties

```diff
@@ -1,7 +1,12 @@
 #Settings to PivotalTracker - UI
-authenticationBaseURL=https://www.pivotaltracker.com
+
+authentication.baseURL=https://www.pivotaltracker.com
 user=
 password=
+projectName=
+accountName=
+newprojectName=
+newdescriptionName=
```

Comment:
In fact you don't need to put the projectName, accountName in the properties file
Only put in this file common variables in the environment.

you can declare them directly in the tests